docs/guide/ops/launching/index.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Launching Brooklyn
+layout: website-normal
+---
+
+There are several ways to launch applications with brooklyn, and useful configuration options,
+as well as a debug-view web console.
+
+This chapter describes how to launch brooklyn.
+
+
+Startup Configuration
+---------------------
+
+brooklyn can read configuration from a variety of places:
+
+* the file ``~/.brooklyn/brooklyn.properties``
+* ``-D`` defines on the brooklyn (java) command-line
+* shell environment variables
```

Comment:
Does the double-backtick have a different meaning to the single-backtick we normally use in Markdown?